### ISSUE TITLE
Future?? (Idea about AngularJS 2)

### ISSUE THREAD

**Issue by chandu0101** [greencatsoft/scalajs-angular#17]
I just saw angularjs 2 presentation https://www.youtube.com/watch?v=gNmWybAyBHI

summary : 
controllers,DDO(directive def obj),module = RIP  , now they are building  everything on components/DI etc.

Instead of mimc'ing angularjs 1.3 features , how about embarrassing new features ... ? if you have plans to do this i'm ready to help , i only have 40 days scala exp ,but i have plenty of time and interest.

Thank You
Chandra

**Comment by chandu0101** [comment]
sorry dude i can't help , the journey i started with this aim took so many turns and ended up here http://chandu0101.github.io/sjrc/
 https://github.com/chandu0101/scalajs-react-components
 now in love with reactjs .
btw good luck with your project.

**Comment by mdedetrich** [comment]
Just wanted to make a general comment here about AngularJS 2.

I wouldn't worry too much about supporting or even taking into account AngularJS 2. Its changing/moving very rapidly, and its design is incredibly different to AngularJS 1. Angular JS 2 is trying to leverage more of the JS ecosystem rather than overbuilding its own (i.e. using EMCA script modules over Angularjs 1.x modules) and its also pushing more and more code into Web Components

I think at this point its just too much work to deal with it, and we may hit blockers in regards to things like modules

**Comment by mysticfall** [comment]
I agree with @mdedetrich about not hurrying up to catch up with Angular 2 yet.

Personally, I'm more interested in testing the web components for now and planning to start work on it soon, but not necessarily in the context of Angular 2.0 though.

But as it seems to move toward that direction also, I suppose it might help us migrating to Angular 2.0 later also.

**Comment by briantopping** [comment]
How are you guys still feeling about this position a couple of months later? I don't have the experience with Angular to say "it's stable", but I like developing with it so far. The icing on the cake would be avoiding learning typescript idioms and possibly contribute back to ng2 bindings in the process.

**Comment by mysticfall** [comment]
Honestly, I've been too occupied with my current project (which still uses Angular 1.4) recently to spend much time in considering the migration path.

So I'm afraid I don't have anything specific to share with you on this subject yet. But considering the pace with which Angular seems to move towards 2.x, I have little doubt we'll be revisiting this soon.

Sorry I couldn't say anything more than this for now.

**Comment by briantopping** [comment]
Hi thanks for your response. I hope no burden or responsibility was implied, just really excited about the prospects and recognizing that I don't have the skills to be productive in this space. So I thought I'd add one voice if nothing else. 

Best of luck in your current endeavors!

**Comment by breatheeasy86** [comment]
When I was referred to this blog by one of my batch-mates who used to  participate along with me at <a href="http://hadooptraininginhyderabad.co.in/angularjs-training-in-hyderabad/">Angularjs Training In Hyderabad</a> center who is also a genius in the subject helped me learn more the tough subject in easy. Thanks you for the information which is to the point, reliable on this website.
